### ISSUE TITLE
Issues on the main LDAP component article

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony-docs#10763]
We received this issue report via some user support channel.

-----

https://symfony.com/doc/current/components/ldap.html#batch-updating

I think, that documentation about method applyOperations() is wrong

```php
$entryManager->applyOperations($entry->getDn(), array(
new UpdateOpteration(LDAP_MODIFY_BATCH_ADD, 'mail', 'new1@example.com'),
new UpdateOpteration(LDAP_MODIFY_BATCH_ADD, 'mail', 'new2@example.com'),
));
```

1. `UpdateOpteration` -> `UpdateOpteration`
2. Third variable in UpdateOpteration::__constructor must be array or null (if first var === LDAP_MODIFY_BATCH_REMOVE_ALL)
3. (!) Each UpdateOperation with LDAP_MODIFY_BATCH_ADD or LDAP_MODIFY_BATCH_REMOVE method are worked with only one item in values array. If I put in values array more that one element, ldap receive error with code 0 and status `Success`
Don't know why

**Comment by TimoBakx** [comment]
1. This was fixed by #10752.
2. Must be `null` only, which is already in the documentation.
3. Seems like this is something done by the internal LDAP functionality (this is all parsed directly into `ldap_modify_batch()` and then handled by the PHP ldap plugin)

**Comment by javiereguiluz** [comment]
Let's close this as "fixed". About the third issue, as @TimoBakx this must be something internal of PHP, because the official PHP docs allow to pass arrays as the values: https://secure.php.net/manual/en/function.ldap-modify-batch.php Thanks!
